modules/react/common/lib/parts/_brand-assets.ts

```diff
@@ -1,11 +1,31 @@
-const CDN_URI = 'https://design.workday.com/images/';
+// const CDN_URI = 'https://design.workday.com/images/';
 
-export const dubLogoBlue = `<img src="${CDN_URI}ck-dub-logo-blue.svg" alt="">`;
+// export const dubLogoBlue = `<img src="${CDN_URI}ck-dub-logo-blue.svg" alt="">`;
 
-export const dubLogoWhite = `<img src="${CDN_URI}ck-dub-logo-white.svg" alt="">`;
+// export const dubLogoWhite = `<img src="${CDN_URI}ck-dub-logo-white.svg" alt="">`;
 
-export const wdayLogoBlue = `<img src="${CDN_URI}ck-wday-logo-blue.svg" alt="">`;
+// export const wdayLogoBlue = `<img src="${CDN_URI}ck-wday-logo-blue.svg" alt="">`;
 
-export const wdayLogoWhite = `<img src="${CDN_URI}ck-wday-logo-white.svg" alt="">`;
+// export const wdayLogoWhite = `<img src="${CDN_URI}ck-wday-logo-white.svg" alt="">`;
+
+// export const miniWdayLogoBlue = `<img src="${CDN_URI}ck-mini-wday-logo-blue.svg" alt="">`;
```

Comment:
did you mean to comment these out or delete theme?

Reply:
Oh I need to delete them. Thanks for catching that!